app/models/application.rb

```diff
@@ -1,4 +1,12 @@
 class Application < ApplicationRecord
+  validates :name, presence: true
+  validates :street_address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip_code, presence: true
+  validates :description, presence: true
+  validates :status, inclusion: { in: ["In Progress", "Pending", "Accepted", "Rejected"] }
```

Comment:
this looks great too! i do think we should be testing these validations though. we can always add them in later!